pkg/controller/container-runtime-config/container_runtime_config_controller.go

```diff
@@ -232,8 +232,11 @@ func (ctrl *Controller) deleteContainerRuntimeConfig(obj interface{}) {
 			return
 		}
 	}
-	ctrl.cascadeDelete(cfg)
-	glog.V(4).Infof("Deleted ContainerRuntimeConfig %s and restored default config", cfg.Name)
+	if err := ctrl.cascadeDelete(cfg); err != nil {
+		utilruntime.HandleError(fmt.Errorf("couldn't delete object %#v: %v", cfg, err))
```

Comment:
In new code let's please use `errors.Wrapf` like `errors.Wrapf(err, "couldn't delete object %s", cfg.Name)` because in the past I and others have forgotten to do the trailing `%v, err` part.

Reply:
@JAORMX did you see this comment?